src/super_gradients/__init__.py

```diff
@@ -1,9 +1,18 @@
+import logging
 from super_gradients.training import ARCHITECTURES, losses, utils, datasets_utils, DataAugmentation, \
     TestDatasetInterface, SegmentationTestDatasetInterface, DetectionTestDatasetInterface, ClassificationTestDatasetInterface, SgModel, KDModel
 from super_gradients.common import init_trainer, is_distributed
 from super_gradients.examples.train_from_recipe_example import train_from_recipe
 from super_gradients.examples.train_from_kd_recipe_example import train_from_kd_recipe
+from super_gradients.sanity_check import env_sanity_check
+from super_gradients.common.abstractions.abstract_logger import get_logger
 
 __all__ = ['ARCHITECTURES', 'losses', 'utils', 'datasets_utils', 'DataAugmentation',
            'TestDatasetInterface', 'SgModel', 'KDModel', 'SegmentationTestDatasetInterface', 'DetectionTestDatasetInterface',
-           'ClassificationTestDatasetInterface', 'init_trainer', 'is_distributed', 'train_from_recipe', 'train_from_kd_recipe']
+           'ClassificationTestDatasetInterface', 'init_trainer', 'is_distributed', 'train_from_recipe', 'train_from_kd_recipe',
+           'env_sanity_check']
+
+logger = get_logger(__name__)
+
+if logger.isEnabledFor(logging.INFO):
```

Comment:
please fix the logic:
if logging.INFO:
   run and run write output to screen and log
else:
   run and write output to log only